feature/platform/transceiver/zr_inventory_test/README.md

```diff
@@ -0,0 +1,51 @@
+# TRANSCEIVER-7: Telemetry: 400ZR module inventory information. 
+
+## Summary
+
+Validate 400ZR modules report inventory information part number and serial
+number.
+
+## Procedure
+
+*   Plug in the ZR module in the host port and make sure the transceiver 
+    state is enabled and host is able to detect the module.
+*   With the module recognized verify it reports correct inventory
+    information by subscribing ON_CHANGE to the following telemetry paths.
+
+    *   /platform/components/component/state/serial-no
+    *   /platform/components/component/state/part-no
```

Comment:
For inventory there are several other pieces which could be useful (under this same "state" container). I wasn't sure if you planned to track these elsewhere, but specifically I was thinking about:

type
description
mfg-name
mfg-date
hardware-version
firmware-version

Should we add them here?

Reply:
I have now included these here in the same test. Thanks